polynote-frontend/polynote/data/messages.ts

```diff
@@ -4,10 +4,10 @@ import {
     arrayCodec, bool, bufferCodec, Codec, CodecContainer, combined, discriminated, either, float64, int16, int32, int64,
     mapCodec, optional, Pair, shortStr, str, tinyStr, uint16, uint32, uint8
 } from './codec'
+import * as deepEquals from 'fast-deep-equal/es6';
```

Comment:
I added this lib for deep object equality.

Reply:
I didn't notice the `isEqual` on another PR end ended up reimplementing a function for shallow object equality: https://github.com/polynote/polynote/blob/master/polynote-frontend/polynote/ui/util/js_object.ts

Can you replace those calls with `deepEquals` as well? (It's only called in the `plot_editor.ts`)

Also, it might make sense to move `mapValues` from that util to `polynote-frontend/polynote/util/functions.ts`.

Sorry... at the time, I only noticed one of the `util` packages. :/

Reply:
Hi @JD557 thanks for letting me know 😄  I'll clean all these up as part of this big refactor.